pkg/controllerstatus/controllerstatus.go

```diff
@@ -26,6 +26,14 @@ type Simple struct {
 	count   int
 }
 
+type UploaderStatus struct {
```

Comment:
If you're using this to do the type check, set an "Operation" field on summary and give that a specific constant that you check for instead.  Wrapping sucks.